viewing_party/party.py

```diff
@@ -1,23 +1,153 @@
 # ------------- WAVE 1 --------------------
 
+from enum import unique
+from turtle import title
+from tests.test_constants import clean_wave_2_data
+
+
 def create_movie(title, genre, rating):
-    pass
+    movie_info = {}
+    keys = ["title", "genre", "rating"]
+    values = [title, genre, rating]
+    for i in range(len(keys)):
+        if values[i] == None:
+            return None
+        movie_info[keys[i]]=values[i]     
+    return movie_info
+
+def add_to_watched(user_data, movie):
+    for key in user_data: 
+        user_data[key]=[movie]
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    for key in user_data: 
+        user_data[key]=[movie]
+    return user_data

```

Comment:
This passes the tests as they are currently written, but there is a significant bug here.  These functions will overwrite the value for every key in the `user_data` dictionary.  The tests for these functions have only a 'watched' or 'watchlist' key, and both start with an empty list, but imagine if we called either of these functions on a user_data dictionary with 'watched', 'watchlist' and 'friends'.  This would overwrite all of the data with `[movie]`. 
```suggestion
def add_to_watched(user_data, movie):
    user_data['watched'].append(movie)
    return user_data

def add_to_watchlist(user_data, movie):
    user_data['watchlist'].append(movie)
    return user_data

```